src/main/java/interfaceadapters/CourseEvaluator/EvaluatorPresenter.java

```diff
@@ -0,0 +1,16 @@
+package interfaceadapters.CourseEvaluator;
+
+import usecases.CourseEvaluator.EvaluatorOutputBoundary;
+import usecases.CourseEvaluator.EvaluatorResponseModel;
+
+public class EvaluatorPresenter implements EvaluatorOutputBoundary {
+
+    @Override
+    public String  prepareSuccessView(EvaluatorResponseModel grade) {
+        return "You got " + grade.getPercentage() + "%" + " in " + grade.getCourseId();
+    }
```

Comment:
Good ! I think for your case you shouldn't have a prepareFailView because by the time we get to your use case, there shouldn't really be any errors.